src/main/kotlin/org/wfanet/measurement/kingdom/deploy/tools/CreateResource.kt

```diff
@@ -140,6 +140,16 @@ private class CreateMcCreationTokenCommand : Runnable {
 
 @Command(name = "data-provider", description = ["Creates a DataProvider"])
 private class CreateDataProviderCommand : CreatePrincipalCommand() {
+
+  @Option(
+    names = ["--required-duchies"],
+    description =
+      [
+        "The set of duchies externals IDS that must be included in all computations involving this DataProvider"
+      ],
+    required = false,
+  )
+  private lateinit var requiredDuchies: List<String>
```

Comment:
This option was noted as optional, thus a default value is required. It raises runtime error "lateinit var not initialized". Could you please fix it? @Marco-Premier